tests/sqlparser_snowflake.rs

```diff
@@ -1064,3 +1065,25 @@ fn test_snowflake_trim() {
         snowflake().parse_sql_statements(error_sql).unwrap_err()
     );
 }
+
+#[test]
+fn parse_agg_with_order_by() {
```

Comment:
This is testing window functions syntax, so perhaps a better name would be

```suggestion
fn parse_window_functions() {
```